vision/getting-started/image_editing.ipynb

```diff
@@ -1,710 +1,1247 @@
 {
```

Comment:
<p>Add a one-line description below this header. </p><p> </p><p>For example:</p><p>"Before outpainting, you must first prepare an image that has some empty padded space around it that you want to outpaint. The cell will below will first create some padding around an image."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/506/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>Done, slightly differently</p>